### ISSUE TITLE
What's the scale of the people that facenet can handle?

### ISSUE THREAD

**Issue by jiapei100** [timesler/facenet-pytorch#27]
For instance, how accurate it is for 100,000 people within top 5 guesses?

**Comment by timesler** [comment]
The accuracy of the model depends on whether it has been trained on those people. For instance, it can achieve > 99% accuracy on over 10000 faces when it is trained directly on examples of those faces. If using pretrained networks to discriminate between 100,000 faces, the accuracy will of course be lower, but you will have to test it to determine how much.
